target/linux/x86/patches-5.10/011-tune_lzma_options.patch

```diff
@@ -0,0 +1,26 @@
+Index: linux-5.10.16/scripts/Makefile.lib
+===================================================================
+--- linux-5.10.16.orig/scripts/Makefile.lib
++++ linux-5.10.16/scripts/Makefile.lib
```

Comment:
Please ensure file name are consistent with other patches ie a / b style

ex:
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib

Reply:
Just refresh the patches aber adding them, the script will do this automatically.